shame_script.py

```diff
@@ -222,8 +263,13 @@ async def fetch_and_send_tasks():
 async def signup_passthrough(
     interaction: discord.Interaction, user_to_signup: discord.Member
 ):
+    logger.info("Signup command received for user: %s", user_to_signup.name)
     await interaction.response.defer(ephemeral=True, thinking=True)
-    await signup(interaction, user_to_signup, bot)
+    try:
+        await signup(interaction, user_to_signup, bot)
+        logger.info("Signup successful for user: %s", user_to_signup.name)
+    except Exception:
+        logger.exception("Error during signup")
 
 
-bot.run(DISCORD_TOKEN)
+bot.run(DISCORD_TOKEN, log_handler=None)
```

Comment:
log_handler = None removes all logging from discord?

Reply:
It blocks discord from creating it's own log handler, which duplicates messages and has its own formatter